vacuum-advisor/vacuum-simulator.mdx

```diff
@@ -0,0 +1,182 @@
+simulation autovacuums more and see if avoidable growth decreases.
+
+There is a specific insight for this:
+[Bloat - Insufficient VACUUM Frequency](/docs/checks/vacuum/insufficient_vacuum_frequency).
+
+### Limitation
+
+* The simulation data tends to trigger autovacuum earlier than the actual
+  triggered timing, especially with tables with lots of UPDATEs.
+  Occasionally, UPDATEs operations don't create dead rows, or even decrease dead
+  rows when "mini autovacuum" happened with these operations.
+  The current simulation can't simulate this situation well, therefore dead rows
+  simply keep increasing with UPDATEs and trigger autovacuum earlier.
+* The avoidable growth graph may over report avoidable growth size due to the
+  limitation of the simulation.
+  You can check the Estimated Table Bloat panel of Schema Statistics
+  VACUUM/ANALYZE Activity page for the current estimated table bloat to access
+  the actual impact.
+
+
+## Triggered by freeze age
+
+To prevent transaction ID wraparound, Postgres runs autovacuums periodically to
+freeze older rows. We also say this as "freeze old transaction IDs",
+You can learn more about freezing in the
+[What is freezing?](/docs/vacuum-advisor/what-is-freezing) page.
+
+![Triggered by freeze age](vacuum_simulator_freezing.png)
+
+When either the age of the oldest unfrozen transaction ID or the oldest unfrozen
+multixact ID exceeds `autovacuum_freeze_max_age` or
+`autovacuum_multixact_freeze_max_age` respectively, freeze age autovacuums will
+be triggered. These autovacuums are called anti-wraparound autovacuums.
+
+With VACUUM Simulator, you can tweak `autovacuum_freeze_max_age` and
+`autovacuum_multixact_freeze_max_age` in this section.
+`autovacuum_multixact_freeze_max_age` can be only tweaked if multixact ID is
+used. Lowering these configuration settings triggers autovacuums more, and
+increasing these configuration settings will decrease autovacuums triggered by
+freeze age.
+
+### Limitation
+
+* The simulation is using the data of average transactions (or multixacts) per
+  second (tps).
+  If the tps varies a lot depending on the time of the day or day of the week,
+  the simulation may look a lot different than the actual data.
+* Autovacuums triggered by other causes usually try to freeze old rows.
+  The simulation also freezes them, however, there are cases that freezing won't
+  happen when there are nothing can be frozen, or autovacuum is more focused on
+  cleaning up dead rows.
+  This can cause the transaction age to be reset even when it shouldn't be in
+  the simulation, potentially cause the simulation to trigger less autovacuums
+  by freeze age.
+
+
+## Triggered by inserts
+
+Inserts autovacuum is introduced on Postgres 13.
+With an inserts only table, because it won't generate any dead rows, autovacuums
+won't be triggered by dead rows. It can still be triggered by freeze age, but
+depending on the database workload, it can be a long time until it's triggered.
```

Comment:
I think we should lead with what insert-triggered autovacuum is (or why it exists), rather than what Postgres version introduced it. We should also point out why it's bad to wait a long time to freeze. And I think "append-only" is a more standard term than "inserts only"; we should probably stick to that.

```suggestion
Since an append-only table won't generate any dead rows, autovacuum
won't be triggered by dead rows. It can still be triggered by freeze age, but
depending on the database workload, it can be a long time until it's triggered.
This can turn normal incremental maintenance into a resource-intensive,
long-running task that can impact database performance. To counteract
this, since Postgres 13, autovacuum can be triggered by INSERTs.
```